contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -326,7 +326,23 @@ contract WithdrawManager is WithdrawManagerStorage /* , IWithdrawManager */ {
           delete ownerExits[keccak256(abi.encodePacked(_token, currentExit.owner))];
         }
 
-        IDepositManager(registry.getDepositManagerAddress()).transferAmount(_token, exitOwner, currentExit.receiptAmountOrNFTId);
+        address depositManager = registry.getDepositManagerAddress(); // TODO: make assembly call and reuse memPtr
+        uint256 amount = currentExit.receiptAmountOrNFTId;
+        uint256 _gas = gasLimit; // can't read gloablvars in asm
```

Comment:
I think `_gas` should be able to do with less gas than `gasLimit` - (which denotes the gas quota for the entire while block)

Also, account for the gas needed for `delegateCall` in `depositManager`

Reply:
Hardcode _gas value here, I think 150k